packages-ts/integration-eqlabs-multisig/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@chainlink/starknet-integration-tests-eqlabs-multisig",
+  "version": "1.0.0",
+  "description": "Multisig integration test",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "npx hardhat test"
+  },
+  "devDependencies": {
+    "@nomiclabs/hardhat-ethers": "^2.0.5",
+    "@chainlink-dev/starknet-multisig": "^0.1.0",
```

Comment:
Deprecated!

Please double check from Lauri, but I think this is the one we should be using: https://www.npmjs.com/package/starsign-multisig

Reply:
We'll switch in a follow-up, starsign-multisig doesn't contain pre-compiled artifacts